avocado/plugins/multiplex.py

```diff
@@ -69,17 +69,16 @@ def configure(self, parser):
                                  help="Show the inherited values")
 
     def _activate(self, args):
-        # Extend default multiplex tree of --env values
+        # Extend default multiplex tree of --mux-inject values
         for value in getattr(args, "mux_inject", []):
```

Comment:
Suggestion: the `value` variable name used here is then mentioned later as a parameter of `data_inject`.  Maybe using a different name would makes things automatically clear.